pkg/kubernetes/subsetdiff.go

```diff
@@ -0,0 +1,131 @@
+package kubernetes
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/objx"
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/sh0rez/tanka/pkg/util"
+)
+
+type difference struct {
+	live, merged string
+}
+
+func (k Kubectl) SubsetDiff(y string) (string, error) {
+	//          is     should
```

Comment:
?

Reply:
old comment, was 
```go
//  is     should
map[string]string{}
```
before

Reply:
What does is and should mean?

Reply:
`is` is the current state of the cluster, `should` is the local config (how it should be)